### DISCUSSION TITLE
Using secure-env on typescript

### DISCUSSION THREAD

**Discussion by mhhabil** [kunalpanchal/secure-env#27] [General]
hello @kunalpanchal, 
I am so excited with secure-env, but I cannot use that on typescript. is there any different modules that I have to install?
Please reply this @kunalpanchal.

**Comment by kunalpanchal** [comment]
Hey @mhhabil, Glad that you decided to use secure-env for your project. I am afraid though right now secure-env doesn't have types support. You can sure add a declaration file to your project for now and use it as such. You can find more information about that here: https://medium.com/@steveruiz/using-a-javascript-library-without-type-declarations-in-a-typescript-project-3643490015f3 (Plan C). If not please feel free to create a PR as I am not available for this month to contribute to the project due to personal reasons. If thats not the case then I would pick up this issue and resolve it sometime in November 2021. There is a open issue for the same: https://github.com/kunalpanchal/secure-env/issues/23
